### ISSUE TITLE
Swapped meaning in translation of "plain text" and "links"

### ISSUE THREAD

**Issue by FajnB** [igorlogius/copy-tabs#6]
**Describe the bug**
(I tested this only in Czech language)
When selecting a option to save cards, the first option says to copy all cards as links, but the outcome is, that it copies the cards as plain text.
Second option says, to copy all cards as plain text, but the cards are copied as links.
The same goes for selected cards.

It seems, that the translation was just swapped, as the grammar is correct, but the meaning is swapped between the two.
Also, the same swapped translations are in the settings of the add-on in default setting for copying.

**To Reproduce** (Translation error)
1. Go to 'add-on card'
2. Click on 'Copy all/selected cards as plain text/links'
3. See error

**Expected behavior**
Option saying "Copy all/selected cards as plain text" should copy cards as plain text
Option saying "Copy all/selected cards as links" should copy cards as links

**Screenshots**
Not needed

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version: [Add-on: 1.13.37 ;; Firefox: 114.0.1]

**Smartphone (please complete the following information):**
 - Device: Not tested
 - OS: Not tested
 - Browser: Not tested
 - Version: Not tested

**Additional context**
Might be a good idea to check if this is not swapped in other languages too.

**Comment by igorlogius** [comment]
Hi, first up  thank you for reporing this issue. :+1: 

AFAICS the translations should match up with the actions, see here: https://github.com/igorlogius/copy-tabs/blob/main/_locales/cs/messages.json

I recorded this short screen capture (with an empty new profile (fx 114.0.1) and just having installed copy-tabs) 

https://github.com/igorlogius/copy-tabs/assets/67047467/8dd3e127-6968-430d-978e-cf3f164feebd

And as far as i can tell everything looks fine. 

If your results performing the same actions differ, i would be interested in seeing a screen capture from your side as a comparison. 

Thanks again for helping out.

**Comment by FajnB** [comment]
Hi, I am terribly sorry.
What happened is, that I have misunderstood the meaning of those descriptions.
As I pasted these to my text editor, I didn't see the text being highlighted as hyperlinks. So I thought it just copied the names of those cards.
Now it makes way more sense.

It might be better to rephrase the translation "Copy all/selected cards as text" to "Copy all/selected tabs as URL text only" in Czech "Zkopírujte všechny/vybrané karty pouze jako URL".
But I think it might be just me. I used this great add-on after a very long time and I suppose, that this was the reason I got confused.

Thank you for your support and again I am very sorry for your trouble.
